docs/reference/sql/create.md

```diff
@@ -66,8 +66,9 @@ Users can add table options by using `WITH`. The valid options contain the follo
 
 | Option  | Description  | Value |
 |---|---|---|
-| ttl  | The storage time of the table data  |   String value, such as `'60m'`, `'1h'` for one hour, `'14d'` for 14 days etc. |
-|  regions | The region number of the table  | Integral value, such as 1, 5, 10 etc. |
+| `ttl`  | The storage time of the table data  |   String value, such as `'60m'`, `'1h'` for one hour, `'14d'` for 14 days etc. |
```

Comment:
Is it possible to list all the time units here? For example,
ms - millisecond,
s - second,
m - minute,
h - hour,
d - day,
w - week,
y - year